upload.py

```diff
@@ -40,11 +41,8 @@ def upload_file(file_name):
 
 
 def main(file_name):
-    try:
-        upload_file(file_name)
-        print('File uploaded successfully.')
-    except Exception as e:
-        print('Failed to upload file due to error:', e)
+    upload_file(file_name)
```

Comment:
`file_name` is not mandatory on the argumentParser, shouldn't we make it mandatory? Either by requiring to be mandatory or by defining as a parameter without the `-` thingie

Reply:
Good spot - made it mandatory.